services/QuillLMS/spec/controllers/translated_texts_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TranslatedTextsController, type: :controller do
+  let(:staff_user) { create(:user, role: 'staff') }
+  let(:regular_user) { create(:user) }
+
+  describe 'GET #index' do
+    context 'when user is staff' do
+      before do
+        allow(controller).to receive(:signed_in?) { true }
```

Comment:
This `allow` looks like it's declared in every context, so I think you might want to declare a `before` block with this `allow` higher up in the hierarchy so you only need it once.  There's some similarities with the `current_user` allow as well, but I note that it's not always the same user, so maybe that doesn't need to be moved up.